code/modules/projectiles/projectile/magic.dm

```diff
@@ -447,7 +447,7 @@
 	qdel(chain)
 	. = ..()
 
-/obj/projectile/magic/aoe/fireball
+/obj/projectile/magic/fireball
```

Comment:
Does this even change anything? The explosion values are the same, so it should still be an AOE?

Reply:
This change has nothing to do with the explosion, it's about the projectile itself hitting players to the sides of it.